tests/rptest/tests/partition_movement_test.py

```diff
@@ -144,104 +142,13 @@ def test_empty(self, num_to_upgrade):
         topics = []
         for partition_count in range(1, 5):
             for replication_factor in (1, 3):
-                name = f"topic{len(topics)}"
-                spec = TopicSpec(name=name,
-                                 partition_count=partition_count,
-                                 replication_factor=replication_factor)
-                topics.append(spec)
-
-        for spec in topics:
-            self.client().create_topic(spec)
-
-        for _ in range(25):
-            self._move_and_verify()
-
-    @cluster(num_nodes=4,
-             log_allow_list=PARTITION_MOVEMENT_LOG_ERRORS +
-             PREV_VERSION_LOG_ALLOW_LIST)
-    @matrix(num_to_upgrade=[0, 2])
-    def test_static(self, num_to_upgrade):
-        """
-        Move partitions with data, but no active producers or consumers.
-        """
-        self.logger.info(f"Starting redpanda...")
-        test_mixed_versions = num_to_upgrade > 0
-        install_opts = InstallOptions(
-            install_previous_version=test_mixed_versions,
-            num_to_upgrade=num_to_upgrade)
-        self.start_redpanda(num_nodes=3, install_opts=install_opts)
-
-        topics = []
-        for partition_count in range(1, 5):
-            for replication_factor in (1, 3):
-                name = f"topic{len(topics)}"
-                spec = TopicSpec(name=name,
-                                 partition_count=partition_count,
-                                 replication_factor=replication_factor)
-                topics.append(spec)
-
-        self.logger.info(f"Creating topics...")
-        for spec in topics:
-            self.client().create_topic(spec)
-
-        num_records = 1000
-        produced = set(
-            ((f"key-{i:08d}", f"record-{i:08d}") for i in range(num_records)))
-
-        for spec in topics:
-            self.logger.info(f"Producing to {spec}")
-            producer = KafProducer(self.test_context, self.redpanda, spec.name,
-                                   num_records)
-            producer.start()
-            self.logger.info(
-                f"Finished producing to {spec}, waiting for producer...")
-            producer.wait()
-            producer.free()
-            self.logger.info(f"Producer stop complete.")
+                name = f"topic_{partition_count}_{replication_factor}"
+                self.rpk_client().create_topic(name, partition_count,
+                                               replication_factor)
```

Comment:
it looks like the body of test_empty is replaced with a call to `create_topic` and all of `test_static` is removed. it's not clear how that corresponds to "partition_movement_test: use new verifier"

Reply:
I've just dropped the static test https://github.com/redpanda-data/redpanda/pull/5238#discussion_r951031145

Reply:
ahh got it ok